odoo/addons/base/models/ir_actions_report.py

```diff
@@ -478,7 +478,7 @@ def render_template(self, template, values=None):
         if values is None:
             values = {}
 
-        context = dict(self.env.context, inherit_branding=values.get('enable_editor'))
+        context = dict(self.env.context, inherit_branding=False)
```

Comment:
@mge-odoo I know there were some issue with this param, is this change correct ?

Reply:
There is no more template / editing asset in the reports